new blog post for maxim (#911)
temporalio/documentation: blog/series-b-announcement-maxim.md

---
slug: series-b-announcement-open-letter
title: An Open Letter to the Temporal User Community
author: Maxim Fateev
author_title: CEO
author_image_url: https://avatars.githubusercontent.com/u/1463622
tags:
  - announcement
date: 2022-2-16T00:00:00Z
image: /img/temporal-home-page-feb-2022.png
---

![Temporal home page with series b announcement](/img/temporal-home-page-feb-2022.png)

<!--truncate-->

> Today Temporal Technologies announced the $103M round of financing led by Index Ventures with all our major investors (Sequoia, Amplify, Addition, Madrona) participating.

Reviewer: ```suggestion
Today Temporal Technologies announced the $103M round of financing [led by Index Ventures](https://www.indexventures.com/perspectives/everything-is-a-workflow-our-investment-in-temporal/) with all our major investors (Sequoia, Amplify, Addition, Madrona) participating.
```

- Developers of any skill level and experience can leverage Temporal
- A namespace in Temporal Cloud becomes as reliable, predictable, and commonplace as an S3 bucket
- There is a vivid ecosystem of open-source projects that leverage Temporal as an underlying engine.

The new financing validates our business model and gives us the opportunity to pursue these goals.

How we plan to use this new funding to help you:

- Improve our open source software.  We’ll have one update on this later today if you join our live stream.
- Create more and better public goods for the developer community adding education, conferences, community support to what we already do
- Enhance our cloud service to make it available to everyone and anyone
- Grow the ecosystem of technologies that work with or build on Temporal

How you can help us:

- Spread the word - so many of our best user stories started with one engineering team learning about Temporal by word of mouth
- Participate in the community - we are committed to building out in the open and want your participation and your feedback

If you’re reading this and are new to Temporal:

> - [Sign up](https://temporal.io/) for the live introductory sessions we’re running this week and next

Author: ```suggestion
- [Sign up](https://calendly.com/community-temporal) for the live introductory sessions we’re running this week and next
```

> - Sign up for the live introductory sessions we’re running this week and next
> - Or explore some of the common Temporal use cases and get started

Reviewer: ```suggestion
- [Sign up](https://temporal.io/#react-aria-2) for the live introductory sessions we’re running this week and next
- Or explore some of the [common Temporal use cases](https://temporal.io/use-cases) and get started
```
